tests/index_hotfix_pkgs/channeldata.json

```diff
@@ -0,0 +1,129 @@
+{
```

Comment:
The tests have always modified files in here. Not sure if these indexes are necessary, or if they will be regenerated as we test conda-index.